hott/cubical/square.hlean

```diff
@@ -76,14 +82,14 @@ namespace eq
     square p₁₀ p₁₂ p₀₁ p :=
   by induction r; exact s₁₁
 
-  infix ` ⬝h `:75 := hconcat
-  infix ` ⬝v `:75 := vconcat
-  infix ` ⬝hp `:75 := hconcat_eq
-  infix ` ⬝vp `:75 := vconcat_eq
-  infix ` ⬝ph `:75 := eq_hconcat
-  infix ` ⬝pv `:75 := eq_vconcat
-  postfix `⁻¹ʰ`:(max+1) := hinverse
-  postfix `⁻¹ᵛ`:(max+1) := vinverse
+  infix `⬝h`:75 := hconcat --type using \tr
+  infix `⬝v`:75 := vconcat --type using \tr
+  infix `⬝hp`:75 := hconcat_eq --type using \tr
+  infix `⬝vp`:75 := vconcat_eq --type using \tr
+  infix `⬝ph`:75 := eq_hconcat --type using \tr
+  infix `⬝pv`:75 := eq_vconcat --type using \tr
```

Comment:
Ah, thanks, that wasn't a concious decision but result of a merge (I put the `type using` there, before the spaces were there :wink: ).